intelmq/bots/outputs/udp/output.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+import socket
+import sys
+
+import intelmq.lib.utils as utils
+from intelmq.lib.bot import Bot
+
+
+class UDPBot(Bot):
+
+    def init(self):
+        self.delimiter = self.parameters.field_delimiter
+        self.header = self.parameters.header
+        self.udp_host = self.parameters.ip
+        self.udp_port = self.parameters.port
```

Comment:
Better use `int(self.parameters.port)` here, in case someone's saving it as port (intelmq-managers default type...)